src/components/DeviceInfo/index.jsx

```diff
@@ -147,6 +147,31 @@ const styles = (theme) => ({
       },
     },
   },
+  // IF on mobile
```

Comment:
can you remove all these added comments? you generally don't want comments that rewrite the code into english, the code should be readable on its own

Reply:
Saw you already took off the comments. Thanks for doing that and for the feedback, will note that for next time